plugins/GSdx/GSdx.cpp

```diff
@@ -155,12 +155,14 @@ GSdxApp::GSdxApp()
 	m_gs_aspectratio.push_back(GSSetting(1, "4:3", ""));
 	m_gs_aspectratio.push_back(GSSetting(2, "16:9", ""));
 
-	m_gs_upscale_multiplier.push_back(GSSetting(1, "Custom", ""));
+	m_gs_upscale_multiplier.push_back(GSSetting(1, "Native", ""));
 	m_gs_upscale_multiplier.push_back(GSSetting(2, "2x Native", ""));
 	m_gs_upscale_multiplier.push_back(GSSetting(3, "3x Native", ""));
 	m_gs_upscale_multiplier.push_back(GSSetting(4, "4x Native", ""));
 	m_gs_upscale_multiplier.push_back(GSSetting(5, "5x Native", ""));
 	m_gs_upscale_multiplier.push_back(GSSetting(6, "6x Native", ""));
+	m_gs_upscale_multiplier.push_back(GSSetting(8, "8x Native", ""));
+	m_gs_upscale_multiplier.push_back(GSSetting(9, "Custom", ""));
```

Comment:
It wouldn't display the frame buffer size for custom resolution if it gets set to 0 after the coverity branch merge. https://github.com/PCSX2/pcsx2/commit/2430abed3342ec09c3708011881a00e47e55a928

either way, it shouldn't be a problem since custom res will be removed as you stated.